File: CSharp/StringCalculator.Logic/StringCalculator1.cs
using System;

namespace StringCalculator.Logic
{
    public class StringCalculator1
    {
        public static void Add(string numbers)
        {
            var numbersArray = numbers.Split(',');
            if (numbersArray.Length > 2)
            {
                throw new ArgumentException();
            }
            foreach(var number in numbersArray)
            {
                Int16.Parse(number);
            }
        }
    }
}

File: Java/StringCalculator/src/main/java/StringCalculator1.java

public class StringCalculator1 {

	public static void add(String numbers) {
		String[] numbersArray = numbers.split(",");
		if (numbersArray.length > 2) {
			throw new RuntimeException("Up to 2 numbers seperated by comma (,) are allowed");
		}
		for (String number : numbersArray) {
			Integer.parseInt(number);
		}
	}

}
